edk2toolext/environment/extdeptypes/nuget_dependency.py

```diff
@@ -106,12 +106,15 @@ def _fetch_from_cache(self, package_name):
                 return_string = return_buffer.read()
                 NugetDependency.global_cache_path = return_string.strip().strip("global-packages: ")
 
+        if NugetDependency.global_cache_path is None:
+            logging.info("Could not determine Nuget global packages cache location")
+            logging.info("Check your mono installation")
+            return False
         #
         # If the path couldn't be found, we can't do anything else.
         #
         if not os.path.isdir(NugetDependency.global_cache_path):
-            logging.info(
-                "Could not determine Nuget global packages cache location.")
+            logging.info("Could not determine Nuget global packages cache location.")
```

Comment:
this message is the same as the above and seems wrong

Reply:
seems like it should be...global cache path is not a directory

Reply:
Yeah- fixed